rasa/nlu/classifiers/keyword_intent_classifier.py

```diff
@@ -48,20 +48,20 @@ def train(
         duplicate_examples = set()
         for ex in training_data.training_examples:
             if (
-                ex.text in self.intent_keyword_map.keys()
-                and ex.get(INTENT) != self.intent_keyword_map[ex.text]
+                ex.get(TEXT) in self.intent_keyword_map.keys()
```

Comment:
I think it would be better in the future to such a changes in a separate PR to keep the surface of the PR smaller and more concise